src/traces/carpet/axis_attributes.js

```diff
@@ -284,6 +284,16 @@ module.exports = {
             'If *B*, 1B.'
         ].join(' ')
     },
+    minexponent: {
+        valType: 'number',
+        dflt: 3,
+        min: 0,
+        role: 'style',
+        editType: 'ticks',
```

Comment:
There is no "ticks" `editType` defined for `carpet` plots.
Let's set 
```js
editType: 'calc',
```
here.

Reply:
Thanks, done.